src/main/scala/com.snowplowanalytics.snowplow.micro/SqliteStorage.scala

```diff
@@ -245,22 +243,6 @@ private[micro] object SqliteStorage {
   // Tunable: increase for more coverage, decrease for better performance
   private val COLUMN_STATS_SCAN_LIMIT = 1000
 
-  private def databaseExecutionContext(size: Int): Resource[IO, ExecutionContext] = {
```

Comment:
This duplicated the standard execution context provided by doobie, so I would prefer to use the standard one.